web/src/components/predictions/questions/interval-choice.js

```diff
@@ -18,7 +21,14 @@ function IntervalChoice(props) {
 
   // Updates Firebase with final values
   const updateFirebase = () => {
-    console.log(values);
+    let updates = {};
+    updates[qid] = values;
+    if (new Date(props.date_expired).getTime() > new Date().getTime()) {
```

Comment:
since `new Date(props.date_expired)` is used multiple times in this component, let's just create a dateExpired variable